src/cpp/benders/benders_core/include/common.h

```diff
@@ -24,6 +24,7 @@
 #include <vector>
 
 enum class MasterFormulation { INTEGER, RELAXED };
+enum class SOLVER { BENDERS, OUTER_LOOP, MERGE_MPS };
```

Comment:
new Benders method (not really, let's assume...), also some users would run mergeMPs in a fullRun

Reply:
TODO